pyproject.toml

```diff
@@ -1,6 +1,90 @@
+[project]
+name = "asv"
+description = "Airspeed Velocity: A simple Python history benchmarking tool"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "LICENSE.rst", content-type = "text/plain" }
+requires-python = ">=3.7"
+authors = [
+    { name = "Michael Droettboom", email = "mdroe@stsci.edu" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Environment :: Web Environment",
+    "License :: OSI Approved :: BSD License",
```

Comment:
Is this the equivalent of "BSD 3 clause licence"?

Reply:
see https://github.com/airspeed-velocity/asv/pull/1239#discussion_r1111250857